File: Code/Julia/Problem0004.jl
@time begin     
function euler4()
    max_palindrome = 0
    for x in range(100, 999)
        for y in range(100, 999)
            num = x*y
            if num == parse(Int64, reverse(string(num))) && num > max_palindrome
                max_palindrome = num
            end
        end
    end

    return max_palindrome
end
euler4()
end
print(euler4())

File: Code/Python/Problem0004.py
max_palindrome = 0

for x in range(100, 1000):
    for y in range(100, 1000):
        number = x*y
        
        reversed_number = str(number)[::-1]
        reversed_number = int(reversed_number)
        
        if number == reversed_number and number > max_palindrome:
                max_palindrome = number
            
print(max_palindrome)